app/src/lib/api.ts

```diff
@@ -185,9 +185,13 @@ export class API {
   public async fetchRepository(owner: string, name: string): Promise<IAPIRepository | null> {
     try {
       const response = await this.request('GET', `repos/${owner}/${name}`)
-      return parsedResponse<IAPIRepository>(response)
+      if (response.status === 404) {
```

Comment:
Can we break out the `404` into a constant or even better, move it to the TS equivalent of a static class?